README.md

```diff
@@ -152,51 +142,54 @@ kubectl config use-context minikube
   minikube addons enable ingress
   ```
 
-There are two configurations to deploy Eclipse CHE on the Kubernetes:
- - first one: for each new workspace Eclipse CHE creates separated namespace:
+There are two configurations to deploy Eclipse Che on the Kubernetes:
+ - first one: for each new workspace Eclipse Che creates separated namespace:
     ```bash
       helm upgrade --install che --namespace che ./
     ```
- - second one: Eclipse CHE creates workspace in the same namespace:
+ - second one: Eclipse Che creates workspace in the same namespace:
     ```bash
     helm upgrade --install che --namespace=che --set global.cheWorkspacesNamespace=che ./
     ```
 
-> Info: To delploy multi-user CHE you can use parameter: `-f ./values/multi-user.yaml`. Also you can set ingress domain with help parameter: `--set global.ingressDomain=<domain>`
+> Info: To deploy multi-user Che you can use parameter: `-f ./values/multi-user.yaml`. Also you can set ingress domain with help parameter: `--set global.ingressDomain=<domain>`
 
-> Notice: You can track deploy CHE with help Minikube dashboard:
+> Note: You can track deploy Che with help Minikube dashboard:
   ```bash
   minikube dashboard
   ```
 
-Create new workspace from stack 'CHE 7'. Run workspace. When workspace will be running you will see Theia IDE. Then you can [test che-machine-exec with help che-theia-terminal-extension](#test-che-machine-exec-with-help-eclipse-che-theia-terminal) and [test che-machine-exec with help che-theia-task-plugin](#test-che-machine-exec-with-help-che-theia-task-plugin)
+Create an Eclipse Che 7.x workspace using the default Theia IDE. Then you can [test che-machine-exec with help che-theia-terminal-extension](#test-che-machine-exec-with-help-eclipse-che-theia-terminal) and [test che-machine-exec with help che-theia-task-plugin](#test-che-machine-exec-with-help-che-theia-task-plugin)
 
-## Prepare Eclipse CHE to deploy
+## Build Eclipse Che Assembly
 
-> Requiements: installed java 8 or higher and maven 3.3.0 or higher
+> Requiements: installed java 8 or higher and maven 3.5 or higher
 
-First of all clone Eclipse CHE:
+First, clone Eclipse Che:
 
 ```
 $ git clone https://github.com/eclipse/che.git ~/projects/che
 ```
 
-For test purpose it's not nessure build all Eclipse CHE, build 'assembly-main' maven module is pretty enough:
+You can save time by simply building the `assembly-main` module, rather than the whole Eclipse Che project.
```

Comment:
Save the time by building the `assembly-main` module, instead of the whole Eclipse Che project.